pom.xml

```diff
@@ -24,24 +24,71 @@
     <version>7</version>
   </parent>
 
-  <groupId>com.google.common.geometry</groupId>
-  <artifactId>s2-geometry-library-java</artifactId>
-  <version>HEAD-SNAPSHOT</version>
+  <groupId>com.google.geometry</groupId>
+  <artifactId>s2-geometry</artifactId>
+  <version>2.0.0</version>
```

Comment:
It may make sense to make the change to 2.0.0 in a separate commit (which you could then tag, as you'd said), keeping the version as HEAD-SNAPSHOT in main branch (as we've been doing [for projects like Guava](https://github.com/google/guava/blob/3381ffa023b1937fceb20977aef68aaf5ce84b21/pom.xml#L9)).

But we can always change to 2.0.0 now to keep things simple and then change back to HEAD-SNAPSHOT later, too.

(The only potential drawback to HEAD-SNAPSHOT that we've heard of so far is discussed in internal bug 195414312. In it, I note that "HEAD-SNAPSHOT" is currently rejected by the Java module system -- which s2-geometry doesn't use and [probably doesn't need to](https://github.com/google/guava/issues/2970#issuecomment-658246719), but it's something to be aware of.)

Reply:
It'll be particularly important to keep release branches to ensure client stability while giving ourselves a clean merge target for any urgent fixes or security issues, at least until the API evolution halts. In the meantime, let's keep this as *some* kind of snapshot label to avoid blessing this as any stable version.

Reply:
Eric, if you agree with Chris, I am reverting the version to HEAD-SNAPSHOT and I will set it to 2.0.0 in a separate commit.